dev/com.ibm.ws.cdi.extension_fat/fat/src/com/ibm/ws/cdi12/suite/FATSuite.java

```diff
@@ -53,4 +71,14 @@ public static void generateHelpFile() {
         FatLogHandler.generateHelpFile();
     }
 
+    @AfterClass
+    public static void transformUserFeatures() {
+        //If we've finished EE8 then transform javax to Jakarta in preperation for EE9
+        if (RepeatTestFilter.isRepeatActionActive("EE8_FEATURES")) {
+            for (String path : PATHS_TO_BUNDLES) {
+                JakartaEE9Action.transformApp(Paths.get(path));
+            }
+        }
+    }
```

Comment:
I hate this.

Ideally, ensure that two versions of each bundle get built, one for javax and one for jakarta so that both can be deployed at once and the test order can be changed without breaking the tests.

At the very least please put it in a `@BeforeClass` when we're starting the EE9 features, rather than relying on EE9 always following EE8.

Reply:
I really don't like the idea of having two identical bundles that must always be kept in sync, so I moved it to the BeforeClass.

Reply:
Ah, I wasn't intending that we'd have two copies of the code, just that we'd compile it once as the EE8 bundle, then transform it into a separate EE9 bundle.

I'm not entirely sure of the build magic that would be required to make that happen, but it seems like it would be similar to the transformation that we do for product bundles.